### DISCUSSION TITLE
log4j vulnerability: leihs affected?

### DISCUSSION THREAD

**Discussion by MatKDA** [leihs/leihs#1263] [Q&A / Community Support] [answered]
Hello,
I just want to ask if leihs is affected by the log4j vulnerability (https://nvd.nist.gov/vuln/detail/CVE-2021-44228 ...)?
Thank you and regards
Matthias

**Comment by DrTom** [comment, marked as answer]
Short answer: Leihs is not affected. 

Full disclosure: leihs is using an older log4j version which is by the current bug not affected.  But that one hasn't received updates for a while. So we will replace it soon.

**Comment by MatKDA** [comment]
Thank you!
